src/components/daydream-controls.js

```diff
@@ -0,0 +1,204 @@
+var registerComponent = require('../core/component').registerComponent;
+var bind = require('../utils/bind');
+var isControllerPresent = require('../utils/tracked-controls').isControllerPresent;
+
+var DAYDREAM_CONTROLLER_MODEL_BASE_URL = 'https://cdn.aframe.io/controllers/google/';
+var DAYDREAM_CONTROLLER_MODEL_OBJ_URL = DAYDREAM_CONTROLLER_MODEL_BASE_URL + 'vr_controller_daydream.obj';
+var DAYDREAM_CONTROLLER_MODEL_OBJ_MTL = DAYDREAM_CONTROLLER_MODEL_BASE_URL + 'vr_controller_daydream.mtl';
+
+var GAMEPAD_ID_PREFIX = 'Daydream Controller';
+
+/**
+ * Vive Controls Component
+ * Interfaces with vive controllers and maps Gamepad events to
+ * common controller buttons: trackpad, trigger, grip, menu and system
+ * It loads a controller model and highlights the pressed buttons
+ */
+module.exports.Component = registerComponent('daydream-controls', {
+  schema: {
+    hand: {default: 'right'}, // This informs the degenerate arm model.
```

Comment:
@machenmusik According to Klaus, the handedness should be on `gamepad.hand` as per https://w3c.github.io/gamepad/extensions.html

I did take a look, and I don't see the property on the gamepad object as of Canary 59.0.3062.0.   Though, we should be looking for it and setting handedness from this.   I would for now even make the default assumption of right-handedness and optionally setting left.